roles/collect/tasks/main.yml

```diff
@@ -48,17 +37,17 @@
   set_fact:
     ocp_insecure_login: '--insecure-skip-tls-verify={{ ocp_validate_cert }}'
     api_host: '{{ ocp_config["ocp_api"] }}'
-    api_path: 'api/v1/namespaces/{{ ocp_metering_namespace }}'
+    api_path: '{{ ocp_config["metering_api"] }}/api/v1/reports/get'
     current_csv_uuid: '{{ collect_csv_uuid }}'  # bug workaround
     format: "&format={{ collect_format }}"
     namespace: "&namespace={{ ocp_metering_namespace }}"
+    temp_config_path: '/tmp/{{ ocp_config["ocp_cluster_id"] }}_{{ collect_ocp_config_file | basename }}'
+
+# When multiple users use the same machine, we can run into file ownership issues here
+# if something goes wrong and the file is not cleaned up
+- name: Move config file to cluster-specific config name
+  command: 'mv /tmp/{{ collect_ocp_config_file | basename }} {{ temp_config_path }}'
```

Comment:
Let's not drop things into `/tmp` directly, please. 

Everything that korekuta works with should be in an application-specific sub-directory (e.g. collect_download_path).

Reply:
This was already happening, not a change I made. Again because of file permissions. If the role fails to execute to the end, this file isn't cleaned up and the next user to come along is blocked trying to write to that file. At least moving it to a cluster specific named version of the file gets around that.